lisa/notifier.py

```diff
@@ -0,0 +1,85 @@
+from dataclasses import dataclass
+from typing import Any, Dict, List, Type
+
+from lisa import schema
+from lisa.util import InitializableMixin, subclasses
+from lisa.util.logger import get_logger
+
+
+@dataclass
+class MessageBase:
+    type: str = ""
+    elapsed: float = 0
+
+
+class Notifier(subclasses.BaseClassWithRunbook, InitializableMixin):
```

Comment:
It's kinda funny that we call this `subclasses` when it's actually one of the superclasses, and `Notifier` is in fact the subclass.

Reply:
Also I thought `Notifier` was going to subclass `Action`?

Reply:
Notifier has no state transition like from start to stop, it will be alive until the test run completed. so it won't be an Action. Any idea about better name? 🤣